src/app/hooks/useTokens.jsx

```diff
@@ -28,9 +28,16 @@ export function useTokens() {
             );
             await token.getSymbol();
             await token.getInfo();
-            if (token.info.accepting !== true) resolve();
+            if (token.info.accepting === false) {
+              resolve();
+              return;
+            }
             await token.getDecimals();
+
+            token.getMaxAdd();
```

Comment:
```suggestion
            await token.getMaxAdd();
```
cause it's an async function

Reply:
well it takes a bit of time to calculate the maxAdd so it was causing a delay between connecting your wallet and the tokens showing up in the drop down menu.

I thought we could remove the await to fix this but looking at it again it can cause issues, so the await is fine for now